packages/bpk-component-graphic-promo/src/BpkGraphicPromo.module.scss

```diff
@@ -0,0 +1,256 @@
+    }
+
+    &__strapline {
+      @include bpk-subheading;
+    }
+  }
+
+  @include breakpoint-portrait-medium {
+    &__sponsor-label {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-heading-3;
+    }
+
+    &__kicker {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-subheading;
+    }
+
+    &__headline {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-hero-5;
+    }
+
+    &__strapline {
+      @include bpk-body-longform;
+    }
+  }
+
+  @include breakpoint-portrait-small {
+    &__sponsor-label {
+      margin-bottom: bpk-spacing-sm();
+
+      @include bpk-label-1;
+    }
+
+    &__kicker {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-label-1;
+    }
+
+    &__headline {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-heading-2;
+    }
+
+    &__strapline {
+      @include bpk-body-default;
+    }
+  }
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+
+    @include breakpoint-landscape {
+      justify-content: center;
+
+      &--start {
+        align-items: start;
+      }
+
+      &--center {
+        align-items: center;
+      }
+
+      &--end {
+        align-items: end;
+      }
+    }
+
+    @include breakpoint-portrait {
+      justify-content: space-between;
+
+      &--inverted {
+        flex-direction: column-reverse;
+      }
+    }
+
+    @include breakpoint-landscape-desktop {
+      height: pxtorem(400);
+
+      @include bpk-margin-leading(bpk-spacing-xxxl(), false);
+      @include bpk-margin-trailing(bpk-spacing-xxl(), false);
+    }
+
+    @include breakpoint-landscape-tablet {
+      height: pxtorem(400);
+
+      @include bpk-margin-leading(bpk-spacing-xxl(), false);
+      @include bpk-margin-trailing(bpk-spacing-xl(), false);
+    }
+
+    @include breakpoint-portrait-large {
+      height: pxtorem(933);
+      padding: bpk-spacing-xl();
+    }
+
+    @include breakpoint-portrait-medium {
+      height: pxtorem(682);
+      padding: bpk-spacing-xl();
+    }
+
+    @include breakpoint-portrait-small {
+      height: pxtorem(500);
+      padding: bpk-spacing-lg();
+    }
+  }
+
+  &__sponsor-content,
+  &__promo-content {
+    display: flex;
+    flex-direction: column;
+
+    @include breakpoint-landscape {
+      &--start {
+        align-items: start;
+        text-align: start;
+      }
+
+      &--center {
+        align-items: center;
+        text-align: center;
+      }
+
+      &--end {
+        align-items: start;
+        text-align: start;
+      }
+    }
+
+    @include breakpoint-landscape-desktop {
+      width: pxtorem(360);
+    }
+
+    @include breakpoint-landscape-tablet {
+      width: pxtorem(320);
+    }
+  }
+
+  &::after {
+    display: none;
+  }
+
+  &__sponsor-logo {
+    width: pxtorem(148);
+  }
+
+  &__cta {
+    // Remove the following 2 lines once the new Button style is reasdy in Backpack. (ST-4135)
+    background: $bpk-color-white;
+    color: black; /* stylelint-disable-line */
```

Comment:
I guess you've already checked that we don't have a black in our Backpack colours?